pkg/cloudprovider/aws/instancetypes.go

```diff
@@ -189,3 +190,11 @@ func (p *InstanceTypeProvider) CacheUnavailable(ctx context.Context, fleetErr *e
 func UnavailableOfferingsCacheKey(capacityType string, instanceType string, zone string) string {
 	return fmt.Sprintf("%s:%s:%s", capacityType, instanceType, zone)
 }
+
+func InstanceTypesCacheKey(provider *v1alpha1.AWS) string {
```

Comment:
I think we can make this private
```suggestion
func instanceTypesCacheKey(provider *v1alpha1.AWS) string {
```

Reply:
ugh, yes correct.